Dockerfile

```
@@ -0,0 +1,12 @@
+FROM golang:1.12 as builder
+
+WORKDIR /go/src/github.com/cloudflare/cloudflared/
+COPY . .
+RUN apt-get update && apt-get install -y --no-install-recommends upx
+RUN make cloudflared
+RUN upx --no-progress cloudflared
+
+FROM grc.io/distroless/base
```

I think this is a typo..
```suggestion
FROM gcr.io/distroless/base
```

---

Thanks for pointing that out. I managed to add some noise while modifying the commit message with `git commit --amend`. Fixed.